packages/core/admin/admin/src/content-manager/utils/schema.js

```diff
@@ -55,7 +55,12 @@ const getAttributes = (data) => get(data, ['attributes'], {});
 const createYupSchema = (
   model,
   { components },
-  options = { isCreatingEntry: true, isDraft: true, isFromComponent: false }
+  options = {
+    isCreatingEntry: true,
+    isDraft: true,
+    isFromComponent: false,
+    disabledIsJSONTest: false,
```

Comment:
ok I guess we will have to do it this way. Maybe we can find something better in the future. I don't like the name though. What do you think of `isJSONTestDisabled`? I was also considering `isListView` and just use that to determine what to do. What do you think is best?